cermine-impl/src/main/java/pl/edu/icm/cermine/ContentExtractor.java

```diff
@@ -847,20 +844,122 @@ public static void main(String[] args) throws ParseException, AnalysisException,
+                    "Usage: ContentExtractor -path <path> [optional parameters]\n\n"
+                    + "Tool for extracting metadata and content from PDF files.\n\n"
+                    + "Arguments:\n"
+                    + "  -path <path>           path to a directory containing PDF files\n"
+                    + "  -limit <int>            (optional) the number of PDF files to parse starting from the top  default: \"all\"\n"
+                    + "  -workers <int>         (optional) how many workers to run in parallel  default: \"1\"\n"
+                    + "  -outpath <path>        (optional) path to a directory to write the resulting files  default: \"-path\"\n"
+                    + "  -outputs <list>        (optional) comma-separated list of extraction\n"
+                    + "                         output(s); possible values: \"jats\" (document\n"
+                    + "                         metadata and content in NLM JATS format), \"text\"\n"
+                    + "                         (raw document text), \"zones\" (text zones with\n"
+                    + "                         their labels), \"trueviz\" (geometric structure in\n"
+                    + "                         TrueViz format), \"images\" (images from the\n"
+                    + "                         document); default: \"jats\"\n"
+                    + "  -exts <list>           (optional) comma-separated list of extensions of the\n"
+                    + "                         resulting files; the list has to have the same\n"
+                    + "                         length as output list; default: \"cermxml,images\"\n"
+                    + "  -override              override already existing files\n"
+                    + "  -timeout <seconds>     (optional) approximate maximum allowed processing\n"
+                    + "                         time for a PDF file in seconds; by default, no\n"
+                    + "                         timeout is used; the value is approximate because in\n"
+                    + "                         some cases, the program might be allowed to slightly\n"
+                    + "                         exceeded this time, say by a second or two\n"
+                    + "  -configuration <path>	(optional) path to configuration properties file\n"
+                    + "                         see https://github.com/CeON/CERMINE\n"
+                    + "                         for description of available configuration properties\n"
+                    );
+            System.exit(1);
+        }
+
+        final boolean override = parser.override();
+        final Long timeoutSeconds = parser.getTimeout();
+        
+        String path = parser.getPath();
+        final String outpath = parser.getOutPath()+'/';
+        final Map<String, String> extensions = parser.getTypesAndExtensions();
+
+        Long fileLength = Files.list(Paths.get(path)).count();
```

Comment:
Here you should be calculating the number of the PDF files in the directory, and you are calculating the number of entries directly in the input directory instead. So two problems: this does not go recursively into subdirectories, and it calculates everything, not only PDF files.